Modules/System Tests/Rest Client/app.json

```diff
@@ -0,0 +1,54 @@
+{
+  "id": "ae153cbb-ad55-447c-9226-5af3ef57280f",
+  "name": "Rest Client Tests",
+  "publisher": "Microsoft",
+  "version": "24.0.0.0",
+  "brief": "Tests for the Rest Client module",
+  "description": "Tests for the Rest Client module",
+  "privacyStatement": "https://go.microsoft.com/fwlink/?LinkId=724009",
+  "EULA": "https://go.microsoft.com/fwlink/?linkid=2182906",
+  "help": "https://go.microsoft.com/fwlink/?linkid=2206603",
+  "url": "https://go.microsoft.com/fwlink/?LinkId=72401",
+  "dependencies": [
+      {
+        "id": "812b339d-a9db-4a6e-84e4-fe35cbef0c44",
+        "name": "Rest Client",
+        "publisher": "Microsoft",
+        "version": "24.0.0.0"
+      },
+      {
+          "id":  "5d86850b-0d76-4eca-bd7b-951ad998e997",
```

Comment:
This dependency is from the Base App. The System App may not take dependencies on the layers above. I'll see if I can't remove that dependency.

Reply:
Apparently wasn't needed at all.